heartbeat/asterisk

```diff
@@ -306,13 +306,24 @@ asterisk_monitor() {
     #   simply no answer (timeout).
     #   This can also happen if sipsak is run too early after asterisk
     #   start.
-    if [ -n "$OCF_RESKEY_monitor_sipuri" ]; then
+   
+   #To avoid the case where the sipsak check runs before the sip starts at the start action
+    if [ -n "$OCF_RESKEY_monitor_sipuri" ]; then   
         ocf_run sipsak -s "$OCF_RESKEY_monitor_sipuri"
```

Comment:
Can you define the command in `cmd=` or similar name and run that with ocf_run in both instances?

That should make it easier to see that they're the same and update in the future.

Reply:
sure